config/default.json

```diff
@@ -19,8 +19,13 @@
       { "period": "5 minutes", "limit": 0 }
     ]
   },
+  "hmac_key": "YOU MUST CHANGE ME",
```

Comment:
does this need to have a more verbose name? what if we have keys for other stuff in the future?

Reply:
Fwiw, that's why I went for a generic name. If we need to HMAC other data in this repo, can't it just be hashed with the same key?

As long as the key used in prod remains secret, I don't see any benefit to having different keys for various data that needs hashing. And using one key just means there's a little bit less config to manage, which seems like a (small) win.